packages/back-end/src/validators/metric-time-series.ts

```diff
@@ -0,0 +1,88 @@
+import { z } from "zod";
+
+const metricTimeSeriesValue = z
+  .object({
+    value: z.number(),
```

Comment:
How to do this transparently so we store the condensed versions (v, d, e) and save space while keeping the same DX?